pkg/artifacts/helm.go

```diff
@@ -181,21 +189,19 @@ func PullChart(chartURL, version string, destDir string, opts ...RegistryClientO
 
 // PushChart pushes the local chart tarFile to the remote URL provided
 func PushChart(tarFile string, pushChartURL string, opts ...RegistryClientOption) error {
-	cfg := &action.Configuration{}
 	reg, err := getRegistryClientWrap(NewRegistryClientConfig(opts...))
 	if err != nil {
 		return fmt.Errorf("missing registry client: %w", err)
 	}
-	cfg.RegistryClient = reg.client
-	client := action.NewPushWithOpts(action.WithPushConfig(cfg))
 
+	client := action.NewPushWithOpts(
+		action.WithPushConfig(&action.Configuration{RegistryClient: reg.client}),
+	)
 	client.Settings = cli.New()
 
-	if _, err := client.Run(tarFile, pushChartURL); err != nil {
-		return fmt.Errorf("failed to push Helm chart: %w", err)
```

Comment:
It looks like you swallowed this error message with your change, i.e. if we merge this PR we won't be showing the user an actual human friendly message.

Reply:
It stuttered before this change, see this example:
```
Failed to push Helm chart: failed to push Helm chart: failed to do request: Head "https://harbor.juan.ariza/v2/charts/grafana/blobs/sha256:ff28b728fcd253503bd27c8dbb0a765d4f89a2b41d88c5991219d8cfec1389b6": tls: failed to verify certificate: x509: “harbor.juan.ariza” certificate is not trusted
```